src/R/Editor/Impl/Completion/Definitions/RCompletionContext.cs

```diff
@@ -118,28 +96,22 @@ public static bool IsCaretInLibraryStatement(ITextView textView)
                     end -= line.Start;
 
                     string s = line.GetText().Substring(start, end - start);
-                    if (s == "library" || s == "require")
-                    {
+                    if (s == "library" || s == "require") {
                         return true;
                     }
                 }
-            }
-            catch (Exception) { }
-        
+            } catch (Exception) { }
```

Comment:
OK. This is probably old protection from random editor exceptions which used to happen years ago.